build-tools/helm/spark-kubernetes-operator/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+################################################################################
```

Comment:
Please remove this because this repository doesn't use this style for ASF license.